python/mlc_llm/compiler_pass/dispatch_kv_cache_creation.py

```diff
@@ -51,7 +51,8 @@ def extract_creation_args(func: relax.Function) -> Dict[str, Any]:
         "rope_scaling": json.loads(args[10].value),
         "rope_ext_factors": args[11],
         "rotary_dim": args[12].value.value,
-        "dtype": args[13].value,
+        "enable_disaggregation": bool(args[13].value.value),
```

Comment:
Is it safe to remove `enable_disaggregation`? The argument is not propagated after this pass

Reply:
No, this one is necessary. It's part of `kwargs` and will be passed along https://github.com/mlc-ai/mlc-llm/blob/fffb785073b740af312450b6b7ed6b1fd5b74b97/python/mlc_llm/compiler_pass/dispatch_kv_cache_creation.py#L103-L108